test/views/ChoiceViewSpec.scala

```diff
@@ -87,7 +89,11 @@ class ChoiceViewSpec extends ViewSpec with ChoiceMessages with CommonMessages {
 
       """.stripMargin)
 
-      val supplementaryAppConfig = new AppConfig(Configuration(supplementaryConfig), Environment.simple())
+
+      val conf: Configuration = Configuration(supplementaryConfig)
+      val runMode: RunMode = new RunMode(conf, Test)
+      val servicesConfig = new ServicesConfig(conf, runMode)
+      val supplementaryAppConfig = new AppConfig(conf, Environment.simple(), servicesConfig, "AppName")
```

Comment:
Just wondering... Can we use something like FakeAppConfig? And have a method that would build and return this config?